GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -259,12 +258,30 @@ void Renderer::frustum_cull(Scene* scene,
 
         // Only push those scene objects that are inside of the frustum
         if (!is_inside) {
-            (*it)->set_in_frustum(false);
+            scene_object->set_in_frustum(false);
+            continue;
+        }
+
```

Comment:
Okay. The camera distance value calculated here can be used for multipurposes, like object ordering, in the future. May be some comments in the codes useful?